netlify.toml

```diff
@@ -1 +1,2 @@
-NODE_OPTIONS="--max-old-space-size=16384"
\ No newline at end of file
+NODE_OPTIONS="--max-old-space-size=16384"
+command = "CI= npm run build"
```

Comment:
we are using yarn not npm

shouldn't this be :

`command = "CI= yarn install && yarn build"`